src/includes/control.h

```diff
@@ -111,6 +111,9 @@ class control_conn_t : private service_listener
     
     // Buffer for outgoing packets. Each outgoing back is represented as a vector<char>.
     list<vector<char>> outbuf;
+    // Output buffer size. functions will increase this value when pushing something into
+    // outbuf and lowering this value when something get removed from outbuf.
```

Comment:
I think a better comment would be "Current output buffer size in bytes", for example. You don't need to say that "functions will increase this value" etc because that is already clear. Comments should describe the purpose of the variable or function and only give further detail if it is really needed.